group_vars/model_ubnt_nanostation_m2_xw.yml

```diff
@@ -0,0 +1,29 @@
+---
+override_target: "ubnt_nanostation-m"
+target: ath79/tiny
```

Comment:
I think this should be ath79/generic (https://firmware-selector.openwrt.org/?version=23.05.4&target=ath79%2Fgeneric&id=ubnt_nanostation-m-xw)